docs/index.md

```diff
@@ -22,16 +22,6 @@ All you need to do for scanning is to specify a target such as an image name of
     <h1 id="demo">Demo</h1>
 </div>
 
-<figure style="text-align: center">
```

Comment:
Try replacing with these movies.
https://user-images.githubusercontent.com/1161307/171013513-95f18734-233d-45d3-aaf5-d6aec687db0e.mov
https://user-images.githubusercontent.com/1161307/171013917-b1f37810-f434-465c-b01a-22de036bd9b3.mov

Reply:
I've replaced it, but i ran into a problem for Firefox, maybe should we convert these movies to gif files?

Reply:
Let's keep it and see how many users will have trouble with that.